qiskit/algorithms/gradients/lin_comb_estimator_gradient.py

```diff
@@ -44,22 +50,34 @@ class LinCombEstimatorGradient(BaseEstimatorGradient):
     `arXiv:1811.11184 <https://arxiv.org/pdf/1811.11184.pdf>`_
     """
 
-    def __init__(self, estimator: BaseEstimator, **run_options):
+    def __init__(
+        self,
+        estimator: BaseEstimator,
+        derivative_type: DerivativeType = DerivativeType.REAL,
```

Comment:
The above DerivativeType enum is not included the in the Sphinx docs and hence the type of this parameter ends up not being linked in the html so you cannot navigate to the type to find out what it is/more about it. You do detail the values in the docstring below though so the information is there but maybe it would be better to have it come out in html too so its linked.